### PR TITLE
Bring Go current in Travis

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: go
 
 go:
-  - 1.10.x
   - 1.11.x
+  - 1.12.x
```
Comment (shinmog) [inline .travis.yml]: Yeah, adding in 1.12 is a good idea.

Comment (gliptak) [reply .travis.yml]: @shinmog the Go release policy states that https://golang.org/doc/devel/release.html#policy ```1.10``` is out-of-support, hence the removal

**Comment by gliptak** [comment]
@shinmog the Go release policy states that https://golang.org/doc/devel/release.html#policy ```1.10``` is out-of-support, hence the removal

**Comment by shinmog** [comment]
Ah, thanks for the info!

lgtm, then

**Pull request merged**